docs/source/en/model_doc/character_bert.md

```diff
@@ -0,0 +1,98 @@
+
+This model is a version of BERT that allows for token-level inputs and outputs. It does not use a subword vocabulary and instead processes each input token at the character-level, making it more flexible and easy to use as well robust to misspellings.
+
+The abstract from the paper is the following:
+
+*Due to the compelling improvements brought by **BERT**, many recent representation models adopted the **Transformer architecture** as their main building block, consequently inheriting the **wordpiece tokenization system** despite it not being intrinsically linked to the notion of Transformers. While this system is thought to achieve a **good balance between the flexibility of characters and the efficiency of full words**, using predefined wordpiece vocabularies from the general domain is not always suitable, especially when building models for **specialized domains** (e.g., the medical domain). Moreover, adopting a wordpiece tokenization **shifts the focus from the word level to the subword level**, making the models **conceptually more complex** and arguably **less convenient in practice**. For these reasons, we propose CharacterBERT, a new variant of BERT that **drops the wordpiece system altogether and uses a Character-CNN module instead** to represent entire words by consulting their characters. We show that this new model **improves the performance of BERT** on a variety of medical domain tasks while at the same time producing **robust**, **word-level** and **open-vocabulary** representations.*
+
+Tips:
+
+- The model is **token-level** but relies on **characters** internally. This means the tokenization can be a simple **split on whitespace**.
+- The WordPiece vocabulary is replaced with a much smaller character (byte) vocabulary. By encoding all texts in **UTF-8**, any characters from any langage can fit in a byte vocabulary of size **28=256 + special tokens**.
+- Instead of **token ids**, the model requires **tokens** to be represented as **lists of character ids**. Therefore, the input shape changes from `(batch_size, sequence_length)` to `(batch_size, sequence_length, token_length)`.
+
+This model was contributed by [helboukkouri](https://huggingface.co/helboukkouri).
+The original code can be found [here](https://github.com/helboukkouri/character-bert).
+
+
+## CharacterBertConfig
+
+[[autodoc]] CharacterBertConfig
+    - all
+
+## CharacterBertTokenizer
+
+[[autodoc]] CharacterBertTokenizer
+    - build_inputs_with_special_tokens
+    - get_special_tokens_mask
+    - create_token_type_ids_from_sequences
+    - save_vocabulary
+
+## CharacterBert specific outputs
+
+[[autodoc]] models.character_bert.modeling_character_bert.CharacterBertForPreTrainingOutput
+
+[[autodoc]] models.character_bert.modeling_tf_character_bert.TFCharacterBertForPreTrainingOutput
+
+[[autodoc]] models.character_bert.modeling_flax_character_bert.FlaxCharacterBertForPreTrainingOutput
```

Comment:
I think these lines need to be removed to make the doc builder pass! CharacterBERT doesn't have FLAX or TF ports, so these classes don't exist, and that's causing the doc builder to fail.
```suggestion

```

Reply:
Done!